module/platform/Webots/src/Webots.cpp

```diff
@@ -564,6 +585,29 @@ namespace module::platform {
                             actuator_requests.motor_pids.emplace_back(
                                 MotorPID(servo.name, {servo.p_gain, servo.i_gain, servo.d_gain}));
                         }
+
+                        // Set the terminate command if the flag is set to terminate the simulator, used by the walk
+                        // simulator
+                        if (terminate_simulation) {
+                            std::cout << "setting terminate on ActuatorRequests to send" << std::endl;
+                            actuator_requests.optimisation_command.command =
+                                OptimisationCommand::CommandType::TERMINATE;
+                            terminate_simulation = false;
+                        }
+
+                        // Set the reset command if the flag is set to reset the simulator, used by the walk simulator
+                        if (reset_simulation_world) {
+                            std::cout << "setting RESET_WORLD on ActuatorRequests to send" << std::endl;
```

Comment:
```suggestion
                            log<NUClear::DEBUG>("Setting RESET_WORLD on ActuatorRequests to send.");
```
Also sounds a bit odd.